entry/src/entry.rs

```diff
@@ -439,13 +439,10 @@ pub fn start_verify_transactions(
         .sum();
 
     // Use the CPU if we have too few transactions for GPU sigverify to be worth it.
-    // Based on the CPU-to-GPU crossover point of 64 in sigverify::ed25519_verify also
-    // taking into account the fact that we have some additional processing
-    // we need to do before we can call sigverify::ed25519_verify
     // TODO: make this dynamic, perhaps based on similar future heuristics
     // to what might be used in sigverify::ed25519_verify when a dynamic crossover
     // is introduced for that function (see TODO in sigverify::ed25519_verify)
-    if api.is_none() || skip_verification || num_transactions < 128 {
+    if api.is_none() || skip_verification || num_transactions < 2048 {
```

Comment:
Have you run the numbers on how many transactions are usually verified at a time on mainnet-beta? I don't think it's this high

Reply:
Switched to a less conservative crossover point. Unit benches on just this function showed about equal perf between the CPU and GPU paths at about 512 transactions. But that's not taking into account that on a real validator, the CPU will likely be doing other things, and with the GPU we have the advantage of being able to do this verification asynchronously, so a crossover point of 512 should still give some benefit. Will try to get some numbers on typical real-world batch sizes.

Reply:
That sounds reasonable to me, thanks!